renku/core/commands/format/dataset_files.py

```diff
@@ -29,13 +32,48 @@ def tabular(client, records, *, columns=None):
     if not columns:
         columns = 'added,creators,dataset,full_path'
 
+    if 'size' in columns.split(','):
+        _get_lfs_file_sizes(client, records)
```

Comment:
This is assuming all dataset files are in LFS - what if that isn't the case?

Reply:
It shows nothing for the size field when the file is not tracked by LFS.

Reply:
Shouldn't it work regardless of how the file is stored?

Reply:
I personally find it confusing to show size on the disk for some files and LFS size (which can be different) for others. But I see your point here. I'll include size on the disk when not in LFS.

Reply:
What do you think about different units for file sizes (e.g. B, KB, MB, ...) which comes from LFS? Should I normalise those to be all MB?

Reply:
Maybe use https://pypi.org/project/humanize/?

Reply:
Now, this displays file size on the disk for those files that are not in LFS. To be compatible with LFS format I've used `humanize` to display humanized file sizes.